yt/loaders.py

```diff
@@ -1457,3 +1461,46 @@ def load_sample(
         loadable_path = loadable_path.joinpath(load_name, specific_file)
 
     return load(loadable_path, **kwargs)
+
+
+# --- Loader for tar-based datasets ---
+def load_archive(
+    fn: str, path: str, ratarmount_kwa: dict = None, *args, **kwargs
```

Comment:
trying to have this type check
```suggestion
    fn: str, path: str, ratarmount_kwa: Optional[Dict] = None, *args, **kwargs
```